frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -90,6 +90,8 @@ comments-AddComment--mention-avatar-alt =
     .alt = User Avatar 
 comments-AddComment--submit-button = <glyph></glyph>
     .title = Submit comment
+comments-Comment--pin-button = { $pinButton }
+    .title = { $pinnedTitle }
```

Comment:
Should we have distinct files for Admin-only strings? Being able to tell if a string  is needed for  users or  admins has  been frequently requested for AMO, for example.

Reply:
I'd leave translations in one file until we make the decision in https://bugzilla.mozilla.org/show_bug.cgi?id=1504710.